crates/infra/utils/src/cargo/public_api.rs

```diff
@@ -1,5 +1,11 @@
+use anyhow::{Context, Result};
```

Comment:
If moving to an `infra` command, I suggest moving the whole `public_api` module to `infra/cli/toolchains` instead, as it won't be needed for all the crates that depend on `infra/utils`.